### PR TITLE
Support deeper subdomains for imple service "Accessing Hostname/IP"

### LINKED ISSUE #306: Simple service "Accessing Hostname/IP" needs to support deeper subdomains
The validation for the "Hostname / IP" field in the accessing configuration does not support hostnames with multiple `.` Characters. The regex needs to be updated to allow deeper subdomains. See screenshot below for example:

Screenshot 2024-04-09 at 11 07 57 AM

### PR DESCRIPTION
**Pull request by rgallettonf**
closes #306

### CODE REVIEW

File: projects/ziti-console-lib/src/lib/services/validation.service.ts [hunk 1]
```diff
@@ -86,6 +86,6 @@ export class ValidationService {
     }
 
     isValidInterceptHost(address) {
-        return /^[^.]+\.[^.]+$/.test(address);
+        return /^(?!\.)(?!.*\.$)([^.]*\.[^.]+)+$/.test(address);
```
Comment (dovholuknf) [inline projects/ziti-console-lib/src/lib/services/validation.service.ts:89]: this should really just be as simple as "contains a period". there are so many rules to get right here, i don't think we should do anything more

Comment (rgallettonf) [reply projects/ziti-console-lib/src/lib/services/validation.service.ts:89]: In that case would it be ok for the following hostnames to be considered valid?
* `.hostname`
* `hostname.`

Comment (rgallettonf) [reply projects/ziti-console-lib/src/lib/services/validation.service.ts]: Updated validation logic.

**Review by lfouts — APPROVED** [review]

**Review by dovholuknf — APPROVED** [review]
it'd be nice to also have a tool tip that tells the user 'why'. do we have that?

**Pull request closed without merging**